src/aktualizr_info/main.cc

```diff
@@ -289,16 +302,28 @@ int main(int argc, char **argv) {
       return EXIT_SUCCESS;
     }
 
+    if (!deviceid_loaded && !tlscred_loaded && ecukeys_loaded) {
```

Comment:
A quite interesting criterion to determine if it's Secondary's DB :)

Reply:
The idea here is that if there is no device id and no tls creds, it means it's for secondary or a database not init successfully, and if the ecu keys existing, it means it's not caused by failure at init time. Yeah, it's a little confused, so, if there is any other options, I like to modify it.

Reply:
It's a bit hacky but it might be the best we can do for now. I'm open to other suggestions.